### ISSUE TITLE
libavbin

### ISSUE THREAD

**Issue by mikemhenry** [pvcraven/arcade#64]
Currently the Linux instructions don't work with Ubuntu 16.04. A few of the packages don't exist in 16.04. Once I get everything working I'll submit a PR for the installation page.

**Comment by mikemhenry** [comment]
Okay, I think I worked out a solution but it requires a manual installation AVbin (not too difficult).

However, I'm getting segfaults if I use pulse audio
If I set pyglet to load the openal module first:
`pyglet.options['audio'] = ('openal', 'pulse', 'silent')`
Things work just fine. This will set pulse to be a fallback and then no audio if that fails as well. Unfortunately this means that on Linux audio will be mono. 

I'll submit a PR with the fix and a PR with the updated instructions.
